dyn/import_dyn_record.go

```diff
@@ -0,0 +1,66 @@
+package dyn
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/nesv/go-dynect/dynect"
+)
+
+func resourceDynRecordImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	results := make([]*schema.ResourceData, 1, 1)
+
+	client := meta.(*dynect.ConvenientClient)
+
+	values := strings.Split(d.Id(), "/")
+
+	if len(values) != 3 && len(values) != 4 {
+		return nil, fmt.Errorf("invalid id for dyn import provided: %s", d.Id())
```

Comment:
This is useful for UX 👍  just one nitpicky question - do you mind mentioning the expected format here? e.g. `invalid id provided, expected format: {type}/{zone}/{fqdn}[/{id}]`

I think we can leave out the "dyn" part and ID passed because that's already obvious from the error message:

> * dyn_record.www (import id: something.hashicorptest.com): import dyn_record.www (id: something.hashicorptest.com): invalid id for dyn import provided: something.hashicorptest.com